### PR TITLE
Update to RMB 33.0.0-pre1 MODCPCT-16

### PR DESCRIPTION
**Pull request by adamdickmeiss**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -156,29 +168,17 @@
...
-                <systemProperty>
-                  <key>project.basedir</key>
-                  <value>${basedir}</value>
-                </systemProperty>
-                <systemProperty>
-                  <key>raml_files</key>
-                  <value>${ramlfiles_path}</value>
-                </systemProperty>
-              </systemProperties>
+              <generateClients>false</generateClients>
```
Comment (julianladisch) [inline pom.xml:181]: `<generateClients>false</generateClients>` is the default. The `<configuration>` section can be removed.

File: pom.xml [hunk 2]
```diff
@@ -56,6 +62,12 @@
       <artifactId>marc4j</artifactId>
       <version>2.9.1</version>
     </dependency>
+    <dependency>
+      <groupId>org.assertj</groupId>
+      <artifactId>assertj-core</artifactId>
+      <version>3.16.1</version>
```
Comment (julianladisch) [inline pom.xml:68]: Can the latest 3.19.0 version been used?

File: pom.xml (hunk 1 above)
Comment (adamdickmeiss) [reply pom.xml:181]: Sure

File: pom.xml (hunk 2 above)
Comment (adamdickmeiss) [reply pom.xml:68]: Yep. Works.

Comment (adamdickmeiss) [reply pom.xml:68]: Yep. Works.

**Review by julianladisch — APPROVED** [review]
Looks good, thanks!

**Pull request merged**